audit/result.go

```diff
@@ -168,16 +168,24 @@ func (r *Result) SetAuditorKey(key []byte) {
 //
 // Zero Result corresponds to incomplete data audit.
 //
-// See also Complete.
+// See also [Result.SetCompleted].
 func (r Result) Completed() bool {
 	return r.v2.GetComplete()
 }
 
+// SetCompleted sets data audit completion flag.
+//
+// See also [Result.Completed].
+func (r *Result) SetCompleted(completed bool) {
```

Comment:
Do you have any use case for `SetCompleted(false)`? Why are we changing the API here?

Reply:
no more relevant https://github.com/nspcc-dev/neofs-sdk-go/pull/590#discussion_r1662714584